collector/settings.mdx

```diff
@@ -409,13 +402,12 @@ Only relevant if you are running your database in Azure using Azure Database for
     <tr>
       <td>azure_ad_tenant_id (<code>AZURE_AD_TENANT_ID</code>)</td>
       <td>[none]</td>
-      <td>The "Directory (tenant) ID" on your application. Only necessary if not using the recommended Managed Identity setup; see these <a href="https://pganalyze.com/docs/log-insights/setup/azure-database#setting-up-authentication-using-azure-ad-application-alternative-to-managed-identity">setup instructions</a>
-      for details</td>
+      <td>The "Directory (tenant) ID" on your application. Only necessary if not using the recommended Managed Identity setup; see these <a href="https://pganalyze.com/docs/log-insights/setup/azure-database#setting-up-authentication-using-azure-ad-application-alternative-to-managed-identity">setup instructions</a> for details</td>
     </tr>
     <tr>
       <td>azure_ad_client_id (<code>AZURE_AD_CLIENT_ID</code>)</td>
       <td>[none]</td>
-      <td>The "Application (client) ID" on your application. Alternatively, when using a Managed Identity, this can also be used to define which identity to use, when a VM has multiple identities assigned.</td>
+      <td>The "Application (client) ID" on your application. Only necessary if not using the recommended Managed Identity setup (see above). Alternatively, when using a Managed Identity, this can also be used to define which identity to use, when a VM has multiple identities assigned.</td>
```

Comment:
Is the "," at the end correct? i.e. shouldn't it be
```suggestion
      <td>The "Application (client) ID" on your application. Only necessary if not using the recommended Managed Identity setup (see above). Alternatively, when using a Managed Identity, this can also be used to define which identity to use when a VM has multiple identities assigned.</td>
```